vars/buildExecute.groovy

```diff
@@ -32,7 +34,11 @@ import static com.sap.piper.Prerequisites.checkScript
     /** For buildTool npm: Execute npm install (boolean, default 'true') */
     'npmInstall',
     /** For buildTool npm: List of npm run scripts to execute */
-    'npmRunScripts'
+    'npmRunScripts',
+    /** Defines if a container image should be created using Cloud Native Buildpacks using the artifact created defined by `buildTool`.
```

Comment:
"a container image" sounds like this would build only one image.

Reply:
See also: https://github.wdf.sap.corp/ContinuousDelivery/piper-library/pull/4966

Reply:
adjusted with the change provided by @pbusko